o-fish-ios/Views/PreboardingFlow/VesselRecordView/LoadingVesselRecordView.swift

```diff
@@ -39,7 +39,7 @@ struct LoadingVesselRecordView: View {
 
     private func loadReports() {
         storedReports = []
-        let predicate = NSPredicate(format: "vessel.name == %@ AND vessel.permitNumber == %@", report.vessel.name, report.vessel.permitNumber)
+        let predicate = NSPredicate(format: "draft == false AND vessel.name == %@ AND vessel.permitNumber == %@", report.vessel.name, report.vessel.permitNumber)
```

Comment:
Split over multiple lines to avoid SwiftLint warning:
```
let predicate = NSPredicate(
            format: "draft == false AND vessel.name == %@ AND vessel.permitNumber == %@",
            report.vessel.name, report.vessel.permitNumber)
```